README.md

```diff
@@ -179,16 +179,38 @@ lintr::lint_package() %>%
 The resulting configuration will contain each currently failing linter and the corresponding number of hits as a comment. Proceed by successively enabling linters, starting with those with the least number of hits. Note that this requires `lintr` 0.3.0.9001 or later.
 
 ## Travis-CI ##
-If you want to run `lintr` on [Travis-CI](https://travis-ci.org) you will need
-to have travis install the package first.  This can be done by adding the
+If you want to run `lintr` on [Travis-CI](https://travis-ci.org) in order to check that commits and pull requests don't deteriorate code style, you will need
+to have Travis install the package first.  This can be done by adding the
 following line to your `.travis.yml`
 
 ```yaml
 r_github_packages:
   - jimhester/lintr
 ```
 
-### Testthat ###
+There are two strategies for getting `lintr` results: 
+
+* either using `lintr::lint_package` as a separate step in your build process (cf [subsection below](#nonfailinglints), which we recommend,
+
+* or setting an unit test (cf [subsection below](#testthat)), which might take a long time to run and hinder development.
+
+In both cases the [lintr-bot](https://github.com/lintr-bot) will add comments
+to the commit or pull request with the lints found and they will also be
+printed on Travis-CI or Wercker.  If you want to disable the commenting you can
+set the environment variable `LINTR_COMMENT_BOT=false`.
+
+### Non-failing Lints <a href="#nonfailinglints" name="nonfailinglints"></a>
```

Comment:
Github automatically generates links for headers, so you don't need to make a new link yourself, the links above should be to `#non-failing-lints` and ``#testthat` below.